File: recommend/src/main/java/com/ue/recommend/util/KLog.java
package com.ue.recommend.util;

import android.util.Log;

/**
 * Created by hawk on 2017/11/27.
 */

public class KLog {
    private static boolean isShowLog;

    public static void init(boolean showLog) {
        isShowLog = showLog;
    }

    public static void e(String tag, String msg) {
        if (isShowLog) {
            Log.e(tag, msg);
        }
    }
}

File: recommend/src/main/java/com/ue/recommend/util/KLog.kt
package com.ue.recommend.util

import android.util.Log

/**
 * Created by hawk on 2017/11/27.
 */

object KLog {
    private var isShowLog: Boolean = false

    fun init(showLog: Boolean) {
        isShowLog = showLog
    }

    fun e(tag: String, msg: String) {
        if (isShowLog) {
            Log.e(tag, msg)
        }
    }
}
